matplotlib2tikz/axes.py

```diff
@@ -198,6 +198,37 @@ def __init__(self, data, obj):
                     assert direction == 'inout'
                     self.axis_options.append('tick align=center')
 
+        # Set each rotation for every label
+        x_tick_labels_rotation = \
+            [label.get_rotation() for label in obj.xaxis.get_majorticklabels()]
+        if any(x_tick_labels_rotation) != 0:
+            # check if every value is the same
+            if all(x == x_tick_labels_rotation[0] for
```

Comment:
I am sorry, but I don't think that this is working. However, I've replaced it with `len(set(x_tick_labels_rotation)) == 1`